lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -39,13 +39,14 @@ def change
       # https://github.com/doorkeeper-gem/doorkeeper/tree/v3.0.0.rc1#custom-access-token-generator
       #
       # t.text     :token,             null: false
-      t.string   :token,             null: false
+      t.string   :token,                  null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.